src/commands/buildCommands/configs/vendorBundlesConfig.js

```diff
@@ -0,0 +1,88 @@
+const ManifestPlugin = require('webpack-manifest-plugin');
+const UglifyJsPlugin = require("uglifyjs-webpack-plugin");
+const webpack = require('webpack');
+const path = require('path');
+
+const paths = require('../paths');
+const env = require('../env');
+const prodPlugins = require('./prodPlugins');
+
+const dllName = "[name]_lib";
+
+const config = {
+	mode: env.properties.isProd ? "production" : "development",
+	entry: {
+		react: [
+			"react",
+			"react-dom",
+			"react-facebook-login",
+			"react-helmet",
+			"react-intl",
+			"react-redux",
+			"react-router",
+			"react-router-dom",
+			"react-waypoint"
+		],
+		vendor: [
+			"autosize",
+			"consolidated-events",
+			"fbjs",
+			"flatpickr",
+			"history",
+			"intl-messageformat",
+			"intl-relativeformat",
+			"prop-types",
+			"qs",
+			"redux",
+			"rison"
+		],
+	},
+	output: {
+		path: paths.output.browser,
+		filename: "[name].[chunkhash].js",
+		hashDigestLength: 8,
+		library: dllName,
+	},
+
+	plugins: [
+		new webpack.EnvironmentPlugin({
```

Comment:
No changes needed to [`webpack.EnvironmentPlugin`](https://webpack.js.org/plugins/environment-plugin/).